webapp/src/components/Modal/BidConfirmationModal/BidConfirmationModal.js

```diff
@@ -1,15 +1,23 @@
 import React from 'react'
 import PropTypes from 'prop-types'
-import { Button, Loader } from 'semantic-ui-react'
+import { Button, Loader, Icon } from 'semantic-ui-react'
+import { eth } from 'decentraland-eth'
+import { env } from 'decentraland-commons'
+import axios from 'axios'
 import { t, T } from '@dapps/modules/translation/utils'
 
 import ContractLink from 'components/ContractLink'
 import { parcelType } from 'components/types'
-import { TOKEN_SYMBOLS } from 'modules/auction/utils'
+import { TOKEN_SYMBOLS, TOKEN_ADDRESSES } from 'modules/auction/utils'
+import { token } from 'lib/token'
 import BaseModal from '../BaseModal'
 
 import './BidConfirmationModal.css'
 
+function formatAmount(amount = 0, digits = 2) {
+  return parseFloat(amount.toFixed(digits), 10).toLocaleString()
```

Comment:
I think we have a `parseFloatWithDecimal` on auction/utils

Reply:
👍